### PR TITLE
TEST: removed unused files for tests

### PR DESCRIPTION
**Pull request by samir-nasibli**
* removed unused files for tests configuration and run, that can possibly confuse developers

### CODE REVIEW

File: tests/run_pytest.py [hunk 1]
```diff
@@ -1,42 +0,0 @@
-import sys
-import pytest
-import os.path
-from yaml import FullLoader, load as yaml_load
-
-
```
Comment (shssf) [inline tests/run_pytest.py:6]: Are you sure this file used nowhere else?
at least, it is used in `README.md`
Please add changes to `README.md`

Comment (samir-nasibli) [reply tests/run_pytest.py:6]: I have not found any files that apply this runner, including configurations, pipelines for CI.
@Alexander-Makaryev Am I right?

Comment (shssf) [reply tests/run_pytest.py:6]: at least, it is used in `README.md`
Please add changes to `README.md`

**Comment by shssf** [comment]
Please add changes to `README.md`

**Comment by samir-nasibli** [comment]
> Please add changes to `README.md`

Done.

**Pull request merged**